server/schemas/typeDefs.js

```diff
@@ -1,9 +1,50 @@
-const { gql } = require('apollo-server-express');
+const { gql } = require("apollo-server-express");
 
 const typeDefs = gql`
   type User {
-    token
-    
+    _id: ID
+    username: String
+    email: String
+    password: String
+    comments: [Comment]
+    order: [Order]
+  }
+
+  type Product {
+    _id: ID
+    name: String
+    price: Float
+    quantity: Int
+    status: String
+    description: String
```

Comment:
if we want to use Schema.types. Mix for description in Mongoose, we can set the type of "JSON" here for the description in graphQL